### PR TITLE
Add Qlik Sense Enterprise URL in help

### PR DESCRIPTION
**Pull request by sublibra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ The catwalk UI doesn't provide login to the Qlik Sense app but as long as there
 
 ![host white list](./images/qmc-whitelist.png)
 
+For connecting to a Qlik Sense Enterprise the websocket URL will be:
+
+`wss://<host>/app/<app-name>`
```
Comment (axelssonHakan) [inline README.md]: This only the case if the default virtual-proxy is used. and the app-name is the docId = GUID for sense server

Comment (sublibra) [reply README.md]: What about:

`wss://<host>/<prefix>/app/<app-guid>`

Where
*  `<prefix>` is a Qlik Sense Enterprise virtual proxy prefix. If the default virtual-proxy is used the `<prefix>` and trailing `/` can be dropped.
* `<app-guid>` is the app's GUID

Comment (peol) [reply README.md]: I'd much rather have support in our websocket inputbox to parse sense client app URLs and _we_ create the websocket URL from that. No instructions, more intuitive, imho.

Comment (sublibra) [reply README.md]: I like that suggestion!

Comment (hrigner) [reply README.md]: Create an issue and close this PR?

Comment (sublibra) [reply README.md]: Yes. #174

**Pull request closed without merging**